### DISCUSSION TITLE
Using the page name as the language name causes issues

### DISCUSSION THREAD

**Discussion by noidar** [i18next/i18next#1804] [Q&A]
Hello,
I am using next-i18next as i18n solution, my page structure is pages[country].
therefore the URL will be http://localhost:3000/de[language]/de[country]/otherpage
router.push('de/otherpage') it swaps de URL to http://localhost:3000/de/otherpage but should be http://localhost:3000/de/de/otherpage
here is the example:
https://github.com/noidar/next-18next-router-problem/blob/master/pages/index.js#L22

Thank you.

**Comment by jamuhl** [comment]
please open questions related to next-i18next in it's repo: https://github.com/i18next/next-i18next
